src/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ ExternalProject_Add(UFS_UTILS
   )
 
 if(NOT CCPP_SUITES)
-  set(CCPP_SUITES "FV3_GFS_v15p2,FV3_RRFS_v1beta")
+  set(CCPP_SUITES "FV3_GFS_v15p2,FV3_RRFS_v1beta,FV3_RRFS_v1alpha")
 endif()
```

Comment:
A decision on whether to keep or remove the RRFS_v1beta SDF for the release is pending, but I'll go ahead and approve this.

Reply:
Please don't merge until the suite has been added to the ufs-weather-model, and the hash in the workflow repo has been updated.

Reply:
@gsketefian, Based on @climbfuji's recent PR in ufs-weather-model, the RRFS_v1beta SDF was removed.  Could you remove that from the list of supported suites from the release?  Then this'll be ready to merge.

Reply:
@JeffBeck-NOAA Done.